### DISCUSSION TITLE
auto generating / dynamic routes

### DISCUSSION THREAD

**Discussion by cpt-n3mo** [rwf2/Rocket#2515] [Questions] [answered]
Good day,

im using rocket of a while now, and i like it allot,
wondering though how one would auto-generate a route, 

for instance in express.js its pretty easy to auto generate some routes from some config object or the like.

but with rocket this seems to be difficult as it depends on the macro system to create the routes.
is there a way to not use the "get, post, put, etc" macros..?  

kind regards

**Comment by SergioBenitez** [comment, marked as answer]
Yes. Please see the `manual-routing` examples in the `examples` directory.
